cms/static/sass/views/_video-upload.scss

```diff
@@ -25,6 +25,10 @@
     cursor:pointer
   }
 
+  .hidden {
+    display: none !important;
```

Comment:
lets get rid of `!important` by swaping `.hidden`'s position with `.show-video-transcripts-wrapper` in stylesheet.